index.js

```diff
@@ -43,6 +43,26 @@
 //.
 //. Sanctuary is designed to work in Node.js and in ES5-compatible browsers.
 //.
+//. ## Installation
+//.
+//. Sanctuary can be installed via npm:
+//.
+//. ```console
+//. $ npm install sanctuary
+//. ```
+//.
+//. Or simply include a script tag in your html document:
+//.
+//. ```html
+//. <script src="https://unpkg.com/sanctuary@latest/dist/sanctuary.min.js"></script>
+//. ```
+//.
+//. You can also try sanctuary in your browser right now in the [Ramda REPL](https://goo.gl/pUU9Ye)
```

Comment:
I'm wary of linking to the Ramda REPL because I don't know how frequently its dependencies are updated, and the relationship between Ramda and Sanctuary has been a source of confusion in the past.

Reply:
That's fair, but it is super helpful for demoing the library.  Any ideas for alternatives that have the same benefits?  Maybe repl.it, or runkit?

Reply:
Would this be better https://repl.it/Ky1q ?

Reply:
I leave this decision to you. If you consider the Ramda REPL the best of the three let's use it but let's first understand the process for updating its dependencies. Linking to a REPL running an old version of Sanctuary might do more harm than good.

Reply:
> Would this be better <https://repl.it/Ky1q> ?

This looks good to me. Does `require('sanctuary')` implicitly refer to the most recently published version?